### PR TITLE
More polishing and explanations

### PR DESCRIPTION
**Pull request by ghost**
- sats and BTC aligned 
- control sum explained
- clearing and polishing (removed unused variables)

image

### CODE REVIEW

File: showTotalLightningBalance.sh [hunk 1]
```diff
@@ -59,76 +58,46 @@ PROFIT_AND_LOSS=$((LN_EARNED_FEES_IN_SATS - ONCHAIN_TX_FEES))
...
+LN_LOCAL_BALANCE_BTC=$(printf %.8f\\n "$((LN_LOCAL_BALANCE))e-8")
+LN_REMOTE_BALANCE_BTC=$(printf %.8f\\n "$((LN_REMOTE_BALANCE))e-8")
+LN_TOTAL_BALANCE_BTC=$(printf %.8f\\n "$((LN_TOTAL_BALANCE))e-8")
 
 ##############################################################################
-# PRINT 
+# PRINT
 ##############################################################################
 
-# turn '0 sats' into '0.000 sats' to keep table nicely formatted
```
Comment (ghost) [inline showTotalLightningBalance.sh:78]: no longer needed as sats and BTC are aligned in a way thats clear, what is realtionship between them
```
    7000000 sats   
 0.07000000 BTC 
         70 %   
```

File: showTotalLightningBalance.sh [hunk 2]
```diff
@@ -59,76 +58,46 @@ PROFIT_AND_LOSS=$((LN_EARNED_FEES_IN_SATS - ONCHAIN_TX_FEES))
...
+echo -e "$(printf %11s "$LN_LOCAL_BALANCE") sats  $(printf %11s "$LN_REMOTE_BALANCE") sats  $(printf %11s $LN_TOTAL_BALANCE) sats"
+echo -e "$(printf %11s "$LN_LOCAL_BALANCE_BTC") BTC   $(printf %11s "$LN_REMOTE_BALANCE_BTC") BTC   $(printf %11s "$LN_TOTAL_BALANCE_BTC") BTC"
+echo -e "$(printf %11s $LN_LOCAL_BALANCE_PERCENTAGE) %     $(printf %11s $LN_REMOTE_BALANCE_PERCENTAGE) %     $(printf %11s $TOTAL_BALANCE_PERCENTAGE) %"
 
 echo -e "\n${YELLOW}ON-CHAIN BALANCE${RESET}"
-echo -e "CONFIRMED\tUNCONFIRMED\tTOTAL"
-echo -e "--------------\t---------------\t---------------"
-echo -e "$ONCHAIN_FUNDS_CONFIRMED sats\t$ONCHAIN_FUNDS_UNCONFIRMED sats\t$ONCHAIN_FUNDS_TOTAL sats"
-echo -e "$ONCHAIN_FUNDS_CONFIRMED_BTC BTC\t$ONCHAIN_FUNDS_UNCONFIRMED_BTC BTC\t$ONCHAIN_FUNDS_TOTAL_BTC BTC"
-echo -e "$ONCHAIN_FUNDS_CONFIRMED_PERCENTAGE%\t\t$ONCHAIN_FUNDS_UNCONFIRMED_PERCENTAGE%\t\t$TOTAL_BALANCE_PERCENTAGE%"
```
Comment (ghost) [inline showTotalLightningBalance.sh:116]: not sure how tabs works in console and how to make it "responsive" I am open to suggestions

**Comment by gcomte** [comment]
I like the new alignment of the sats and BTC representation and I'm okay with the long BTC representation, like this users understand better, what's going here while it's still readable.
The representation does not break up to 99.99999999 BTC, which should be sufficient. At least it is for me ;-P

---

I don't know either how the tab works in the shell. But I think you've solved this problem neatly.

---

The comments at the end are very helpful!

---

We should harmonize the way colors are set, but I'll do that in a later commit.

---

Why the cross sign (✝) ? It might be somewhat misleading. Reminds of death, like dead funds or something.

Other options: two asterisks, a _c_ (as in _control_) or [some varation of it](http://www.i2symbol.com/abc-123/c), like the cents-sign (¢), a hash sign (#), Ⓐ for Audit and Ⓒ for Control Sum, Σ (as in _sum_) or this funny symbol, which like many of my proposals is too big, but it has a _c_ as in _control_ and a _+_ as in _sum_: ⚸

I'll leave it up to you, even if you still think the cross (✝) is the best fit, I'm okay with that.

Selection_176

**Comment by ghost** [comment]
Cross replaced with `**`

**Comment by gcomte** [comment]
Thank you!

Was there a specific reason for why you set the color _cyan_ in a different way than we we did for the other colors?
Because I switched it back such that we do it the same way everywhere: https://github.com/gcomte/ln-gems/commit/16ea399ac55224eab20071221b7d7cff51fdec2e

**Comment by ghost** [comment]
I was trying to use gray color, but I was unsuccessful. I found this by accident and I liked it. I didn't know it is in standard pallet of colors :)

Thanks for fixing it.

**Pull request merged**